### ISSUE TITLE
Doesn't work with YouTube shelf_id

### ISSUE THREAD

**Issue by davidhedlund** [deep-gaurav/MusicPiped#82]
I just published Music for psychedelic therapy: https://www.youtube.com/channel/UC6_LWunY4XtswNBoQ_qLxxA/playlists?view=50&sort=dd&shelf_id=5&view_as=subscriber

I found the word "shelf" in the URI, that's why I included it in the Title for this issue.

It's inconvenient to import the playlists one by one manyally in MusicPiped when all of them can be downloaded if you implement a functionality to support shelf_id in the URI, not just YouTube playlists.

Under Playlists -> Import Playlist, you should change the text "Youtube playlist URL" to "YouTube link" if you implement this.

**Comment by deep-gaurav** [comment]
I'm sorry but I'm using Invidious API to fetch data from YouTube and it currently doesn't support shelf_id

https://github.com/omarroth/invidious/wiki/API

Can possibly raise an issue on Invidious GitHub page if it can be implemented

**Comment by davidhedlund** [comment]
> I'm sorry but I'm using Invidious API to fetch data from YouTube and it currently doesn't support shelf_id
> 
> https://github.com/omarroth/invidious/wiki/API
> 
> Can possibly raise an issue on Invidious GitHub page if it can be implemented

Yes, please, use https://github.com/deep-gaurav/MusicPiped/issues/82 as reference when you submit it, and give me the link here later.

**Comment by davidhedlund** [comment]
https://github.com/deep-gaurav/MusicPiped/issues/83
